logs/logs.go

```diff
@@ -177,8 +177,8 @@ func LogDownload(typ string, statuscode int, p *mirrors.Results, err error) {
 			sameASNum = "same"
 		}
 
-		dlogger.l.Printf("%s %d \"%s\" ip:%s mirror:%s%s %sasn:%d distance:%skm countries:%s",
-			typ, statuscode, p.FileInfo.Path, p.IP, m.Name, fallback, sameASNum, m.Asnum, distance, countries)
+		dlogger.l.Printf("%s %d \"%s\" method:%s ip:%s mirror:%s%s %sasn:%d distance:%skm countries:%s",
+			typ, statuscode, p.FileInfo.Path, method, p.IP, m.Name, fallback, sameASNum, m.Asnum, distance, countries)
```

Comment:
Why not log `method:%s` in other cases, while you're at it? Ie. in the cases below, with `statuscode == 404`, `500` and the final else?